scripts/download_chrome.js

```diff
@@ -38,4 +42,15 @@ async function download() {
 
 download().then(() => {
     console.log(`${browserVersion} downloaded into:`, outputPath);
+
+    // All the chrome files should be in the plugin 'chrome' folder
+    // This is used to set the puppeteer.executablePath
+    const ext = platform === BrowserPlatform.WIN64 ? '.exe' : '';
+    const out = childProcess.execFileSync('find', [`${outputPath}/chrome`, '-type', 'f', '-name', `chrome${ext}`, '-exec', 'dirname', '{}', '\;']);
```

Comment:
This returns an empty string for me and as a result, the whole build fails. Also I'm not sure I get the purposes of this, before the changes the chrome was being downloaded into `${outputPath}/chrome`, which is the built plugin's folder.

Reply:
The thing is in `app.ts` we need to know where `chrome.exe` is and currently it's in `${outputPath}/chrome/<platform>-<buildVersion>/chrome-<patform>`. The goal of this PR is to get rid of these two intermediate folders that are platform/build-specific so we have a generic path that can be used everywhere. 

Before, we used `executablePath()` to get this information but now that it's not working anymore, I couldn't find another way to do this.

Reply:
The folder structure or the file name may be different on Mac, I'll take a look.

Reply:
Ah ok, I see, makes sense.

Reply:
Actually, I think it's [this one](https://github.com/puppeteer/puppeteer/blob/main/packages/browsers/src/launch.ts#L64).